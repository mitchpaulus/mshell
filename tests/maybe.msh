none   (2 +) map str wl
1 just (2 +) map str wl
3 just? 2 + str wl
