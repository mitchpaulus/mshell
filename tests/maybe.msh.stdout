None
Just(3)
5
